src/apps/users/services/core.py

```diff
@@ -71,12 +71,13 @@ async def send_password_recovery(
             "link": (
                 f"https://{settings.service.urls.frontend.web_base}"
                 f"/{settings.service.urls.frontend.password_recovery_send}"
-                f"?key={password_recovery_info.key}&email={user.email}"
+                f"/{user.id}/token/{password_recovery_info.key}?lang=en"
             ),
         }
         message = MessageSchema(
             recipients=[user.email],
-            subject="Password recovery for Mindlogger",
+            subject="Girder for Mindlogger (development instance): "
```

Comment:
I think Girder is not needed

Reply:
Decided to leave it like that for now.
According to this document https://mindlogger.atlassian.net/wiki/spaces/MINDLOGGER1/pages/34832485/US-1.5.1+-+Password+reset+email
described in the third paragraph

Reply:
After talking with Veronica Grigorieva, it was decided to leave everything as in a US-1.5.1 for now.
Later there will be a task to restore beauty in this html.
Please agree.